src/ui-kit/form-controls/text/text.component.ts

```diff
@@ -92,6 +92,12 @@ export class SamTextComponent implements ControlValueAccessor,
    * Sets the title attribute on the input for accessibility
    */
   @Input() public title: string;
+
+  /**
+   * Sets full hint toggle option for label wrapper
+   */
+  @Input() public showFullHint: boolean = false;

```

Comment:
Input for sam-text component to mirror input in sam-label-wrapper. This option is available in sam-label-wrapper, but not provided by sam-text.